server/src/infra/entities/system-config.entity.ts

```diff
@@ -90,6 +92,11 @@ export enum ToneMapping {
   DISABLED = 'disabled',
 }
 
+export enum Colorspace {
```

Comment:
Colour space is two words, not one.

```suggestion
export enum ColorSpace {
```